packages/async-flow/src/types.d.ts

```diff
@@ -0,0 +1,252 @@
+import type { Passable } from '@endo/pass-style';
+import type { Vow } from '@agoric/vow';
+import type { LogStore } from './log-store.js';
+import type { Bijection } from './bijection.js';
+
+export type FlowState =
+  | 'Running'
+  | 'Sleeping'
+  | 'Replaying'
+  | 'Failed'
+  | 'Done';
+
+/**
+ * `T` defaults to `any`, not `Passable`, because unwrapped guests include
```

Comment:
Realize this is an existing comment, but consider an `@internal` directive

Reply:
I don't know how that will render but worth a try

Reply:
What does @internal mean/do?

Reply:
https://typedoc.org/tags/internal/

Reply:
https://github.com/Agoric/agoric-sdk/pull/9690 fixes the lint error. I'm still not sure whether this should be `@internal`.